### PR TITLE
fix: toast content as react node and not mandatory options param

### PR DESCRIPTION
**Pull request by solidovic**

### CODE REVIEW

File: packages/toast/src/ToastDefault.tsx [hunk 1]
```diff
@@ -1,7 +1,10 @@
...
+import { toast, ToastOptions, ToastContent } from 'react-toastify'
 import { TOASTS_DEFAULT_OPTIONS } from './toastsDefaultOptions'
 
-export function ToastDefault(text: string, options: ToastOptions): ReactText {
-  return toast(text, { ...TOASTS_DEFAULT_OPTIONS, ...options })
+export function ToastDefault(
+  content: ToastContent,
+  options?: ToastOptions
+): ReactText {
+  return toast(content, { ...TOASTS_DEFAULT_OPTIONS, ...options })
```
Comment (rkolpakov) [inline packages/toast/src/ToastDefault.tsx:9]: If options object is empty, what would be destructured here?

File: packages/select/src/Select.stories.tsx [hunk 2]
```diff
@@ -45,6 +45,7 @@ const iconsMap = {
 }
 
 export const Icons: Story<SelectProps> = (props) => {
+  // use
```
Comment (rkolpakov) [inline packages/select/src/Select.stories.tsx:48]: Is this comment useful for some reason?

**Review by rkolpakov — CHANGES_REQUESTED** [review]

File: packages/toast/src/ToastDefault.tsx (hunk 1 above)
Comment (solidovic) [reply packages/toast/src/ToastDefault.tsx:9]: options can be undefined. so will get only TOASTS_DEFAULT_OPTIONS

**Review by rkolpakov — APPROVED** [review]

**Review by avsetsin — APPROVED** [review]
LGTM

**Pull request merged**